hrjobcontract/js/build.tmp.js

```diff
@@ -9,7 +9,7 @@
     'common': 'empty:',
     'job-contract/vendor/fraction': 'job-contract/vendor/fraction',
     'job-contract/vendor/job-summary': 'job-contract/vendor/jobsummary',
-    'leave-absences': '/vagrant/hr1703/sites/all/modules/civicrm/tools/extensions/civihr/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences'
+    'leave-absences': '/Users/bmn/buildkit/build/hr17-new/sites/all/modules/civicrm/tools/extensions/civihr/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences'
```

Comment:
What is that? Is "bmn" your username @bmnepali ? how come in the production code we have a developer's personal home path? why it had '/vagrant/hr1703' before?

Reply:
Apologies, this is tmp file so it is OK that it has personal paths. But it must not be committed anyway. It is built in order to create min file at the end and should be deleted. If it is not deleted during gulp, then smth is going wrong.

Reply:
@bmnepali we need to gitignore this file in hrjobcontract

Reply:
@bmnepali was the file removed from the repo?

Reply:
Ignored in  `.gitignore` and removed form repo!